conf/squid/rados/3AZ-cluster.yaml

```diff
@@ -5,6 +5,8 @@ globals:
   - ceph-cluster:
       name: ceph
       node1:
+        networks:
```

Comment:
change the networks to use networks from RHOS-1.

Current networks are from rhos-d

Reply:
created new file, 1 for rhos-d & another for rhos-1. 

Creating 2 files as currently, it's needed to manually deploy cluster for testing.